barcodes_generator_abstract/models/barcode_generate_mixin.py

```diff
@@ -29,14 +29,25 @@ class BarcodeGenerateMixin(models.AbstractModel):
         related="barcode_rule_id.generate_type",
     )
 
+    def write(self, vals):
+        res = super(BarcodeGenerateMixin, self).write(vals)
```

Comment:
```suggestion
        res = super().write(vals)
```

nitpicking : lighter syntax.